examples/lltest.py

```diff
@@ -131,6 +132,8 @@ def parse_args():
                         help='Enable debugging output')
     parser.add_argument('--debug-fuse', action='store_true', default=False,
                         help='Enable FUSE debugging output')
+    parser.add_argument('--aio', choices=('asyncio', 'trio'), default='trio',
```

Comment:
I understand the idea (testing the asyncio code), but I don't like making the event loop configurable for the examples. It makes the code more complicated (and the examples are supposed to be as simple as possible), and from a user point of view doesn't make any sense (if you call a program, you don't care what event loop it uses internally).

I think we should just provide one new example that exclusively uses asyncio.

Reply:
Sounds good to me. Do you have an additional example in mind which would be useful?

Reply:
I'd just rename `lltest.py` to `hello.py`, then copy `hello.py` to `hello_asyncio.py`, make the changes needed for asyncio and be done with it. The current filename is rather confusing for historical reasons.

Reply:
Done in 0f7996ec